tests/dpsdgym/metrics/wasserstein.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+import pandas as pd
+
+def wasserstein_randomization(d1_large, d2_large, iters, downsample_size=100):
+    """
+    Calculate wasserstein randomization test results
+    "We propose a metric based on
+    the idea of randomization inference (Basu, 1980; Fisher, 1935). 
+    Each data point is randomly assigned to one of two
+    data sets and the similarity of the resulting two distributions 
+    is measured with the Wasserstein distance. Repeating this
+    random assignment a great number of times (e.g. 100000 times) 
+    provides an empirical approximation of the distances’
+    null distribution. Similar to the pMSE ratio score we then 
+    calculate the ratio of the measured Wasserstein distance and
+    the median of the null distribution to get a Wasserstein distance 

```

Comment:
Can you give the param doc strings. They make some sense but will be important to understand